meson.build

```diff
@@ -61,4 +61,19 @@ else
     meson.add_dist_script(py, versioneer, '-o', '_version_meson.py')
 endif
 
+if get_option('disable-optimization')
+  add_project_arguments('-DNPY_DISABLE_OPTIMIZATION', language: ['c', 'cpp'])
+  CPU_CONF_BASELINE = 'none'
+  CPU_CONF_DISPATCH = 'none'
```

Comment:
There's a logic behind it, simply any variable starts with `cpu_` can be overrided or unset while any variable starts with `CPU_` should be untouched during life-time of the configuring process, why? because this implementation is separated into two scripts, the issue that second script going to be executed in any order and may override unrelated variable so I had to use `cpu_` as prefix for all its variables.

Reply:
Ah okay, I missed that. That's a good reason, thanks.